includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -966,33 +968,49 @@ public static function print_validation_errors_meta_box( $post ) {
 		} );
 		</script>
 
+		<?php
+		$forced_sanitized_count = 0;
+		foreach ( $validation_errors as $validation_error ) {
+			if ( isset( $validation_error['forced_sanitization'] ) ) {
+				$forced_sanitized_count++;
+			}
+		}
+		?>
+
+		<?php if ( $forced_sanitized_count > 0 ) : ?>
+			<div class="notice notice-info notice-alt inline">
+				<p><?php esc_html_e( 'This site is configured to automatically decide whether or not some validation errors may be accepted. For these errors, you cannot change their status below.', 'amp' ); ?></p>
+			</div>
+		<?php endif; ?>
+
 		<div class="amp-validation-errors">
 			<ul>
 				<?php foreach ( $validation_errors as $error ) : ?>
 					<?php
 					$collapsed_details = array();
 					$term              = $error['term'];
 					$select_name       = sprintf( '%s[%s]', AMP_Validation_Manager::VALIDATION_ERROR_TERM_STATUS_QUERY_VAR, $term->slug );
+					$term_status       = $term->term_group;
```

Comment:
Not sure I see the reason to store this in a variable.

Reply:
You're right. It's no longer needed since the `term_group` is overridden in `get_invalid_url_validation_errors`.